src/components/widgets/Dropdown.js

```diff
@@ -48,35 +43,33 @@ class Dropdown extends Component {
       dropdownStyle.width = width;
     }
 
-    const opts = options.slice();
-    for (let i = 0; i < opts.length; i++) {
-      if (typeof opts[i] === 'string') {
-        opts[i] = {label: opts[i], [valueKey]: opts[i]};
-      }
-    }
+    const opts = options.map(opt =>
+      typeof opt === 'string' ? {label: opt, [valueKey]: opt} : opt
+    );
 
     const dropdownContainerClass = classnames('dropdown-container', {
       'dropdown--dark': this.props.backgroundDark,
-      [this.props.className]: this.props.className,
+      [className]: className,
     });
 
     return (
       <div className={dropdownContainerClass} style={dropdownStyle}>
         <Select
-          backspaceToRemoveMessage={backspaceToRemoveMessage}
           placeholder={placeholder || _('Select an Option')}
-          clearable={clearable}
-          value={value}
+          isClearable={clearable}
+          value={opts.filter(o =>
+            Array.isArray(value) ? value.includes(o.value) : value === o.value
+          )}
           options={opts}
-          searchable={searchable}
+          isSearchable={searchable}
           onChange={this.onChange}
-          multi={multi}
-          optionRenderer={optionRenderer}
-          valueRenderer={valueRenderer}
-          noResultsText={noResultsText || _('No Results')}
-          valueKey={valueKey}
```

Comment:
how about this, not needed anymore?

Reply:
noResultsText = noOptionsMessage
The rest are superseded by `components`